templates/docs/examples/templates/tick-element-comparison.html

```diff
@@ -2,66 +2,137 @@
 {% block title %}Forms / Tick elements{% endblock %}
 
 {% block content %}
-<div class="p-strip is-shallow u-no-padding--top">
-  <div class="row">
-  </div>
-  <div class="row">
-    <div class="col-3">
-      <form action="">
+<!-- <div class="p-strip--dark">
```

Comment:
Is this meant to be commented out? If so, why not removing it?

Reply:
Removed, ready for another look